hier_config/hc_child.py

```diff
@@ -761,11 +752,8 @@ def line_inclusion_test(self, include_tags, exclude_tags):
 
         if include_line:
             set_exclude_tags = set(exclude_tags)
-            if self.tags.intersection(set_exclude_tags):
```

Comment:
Its more readable to avoid doing a negative conditional.  Also, the change gets rid of the return False.  How about this instead:
```Python
if self.tags.intersection(set_exclude_tags):
    return False
return True
```
or a bit less readable but one line

```Python
return not bool(self.tags.intersection(set_exclude_tags))
```